tenant_schemas/models.py

```diff
@@ -6,6 +6,27 @@
 from tenant_schemas.utils import get_public_schema_name, schema_exists
 
 
+class TenantQueryset(models.QuerySet):
+    """
+    QuerySet for instances that inherit from the TenantMixin.
+    """
+    def delete(self):
+        """
+        Make sure we call the delete method of each object in the queryset so
+        that safety checks and schema deletion (if requested) are executed
+        even when using bulk delete.
+        """
+        counter, counter_dict = 0, {}
```

Comment:
This is to be consistent with the Queryset.delete() method that (since django 1.9) returns a tuple, see: https://docs.djangoproject.com/fr/1.11/ref/models/querysets/#delete
It will also work with the django 1.8 implementation that didn't return a value.